mapping_parenting_tech/analysis/prototyping/play_store/get_play_store_apps.py

```diff
@@ -21,694 +22,57 @@
-    """
-    Retrieves ids of Play Store apps related to `seed_app_id` by calling itself recursively.
-
-    Args:
-        seed_app_id: str - the app id of the app of interest
-        depth: int, default = 5 - the depth of recursion. This will increase the number of apps interrogated (and
-        therefore the time taken for the initial call to complete) exponentially
-        current_depth: used for recursion, should be left blank by user
-
-    Returns:
-        a list of app ids
-    """
-
-    app_details = app(seed_app_id, country="gb")
-    similar_apps = app_details["similarApps"] if app_details is not None else list()
-
-    snowball = set()
-    snowball.update(similar_apps)
-
-    if __current_depth < depth:
-        for this_app in similar_apps:
-            snowball.update(app_snowball(this_app, depth, (__current_depth + 1)))
-
-    return list(snowball)
-
-
-# %%
-def save_app_ids(
-    app_list: list, app_category: str, filename: str, output_path: Path = APP_IDS_PATH
-) -> bool:
-    """
-    Saves a list of app ids to a given CSV file. If the file already exists, it is overwritten.
-
-    Args:
-        app_list: list - a list of app ids to be saved
-        app_category: str - the title of the app category being saved, used as the header at the top of the list
-        filename: str - the filename to be used, including extension
-        output_path: Path - destination folder; default is `outputs/data`
-
-    Returns:
-        True if executed successfully
-
-    """
-
-    output_target = output_path / filename
-
-    with open(output_target, "wt", newline="\n") as output_file:
-        csv_writer = csv.writer(output_file)
-        csv_writer.writerow([app_category])
-        # NB can't use writerows() as it will treat each string as a list and split it into comma-separated letters
-        # hence why `app_id` (and `app_category`, above) is encapsulated in square brackets
-        for app_id in app_list:
-            csv_writer.writerow([app_id])
-
-    logging.info(f"App ids saved in {output_target}")
-
-    # if we got here, it's all ok so return True
-    return True
-
-
-# %%
-def load_app_ids(filename: str, file_path: Path = APP_IDS_PATH) -> tuple:
-    """
-    Loads a pickled list of app ids from a given file and returns the list object
-
-    Args:
-        filename: str - file name of pickle file to load within PARENT_DIR
-        file_path: Path - location of the file given in `filename`; default is `outputs/data`
-
-    Returns:
-        str: the name of the app category
-        list: list of app ids
-
-    """
-
-    app_id_list = []
-    with open(file_path / filename, "rt") as id_list:
-        csv_reader = csv.reader(id_list)
-        for app_id in csv_reader:
-            app_id_list.extend(app_id)
-
-    app_category = app_id_list.pop(0)
-    return (app_category, app_id_list)
-
+from mapping_parenting_tech.utils import play_store_utils as psu, logging
```

Comment:
This line threw an error:
```
ImportError: cannot import name 'logging' from 'mapping_parenting_tech.utils'
```

Perhaps you can do:
```suggestion
from mapping_parenting_tech.utils import play_store_utils as psu
from mapping_parenting_tech.utils.play_store_utils import logging
```

Reply:
Think I just needed to import logging separately from `mapping_parenting_tech` (it's not in `utils`). Have updated.